src/App.tsx

```diff
@@ -1,25 +1,56 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect } from "react";
+import Header from "./Design-system/Header";
+import Footer from "./Design-system/Footer";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Home from "./Pages/Home";
+import CharacterDetailsPage from "./Pages/CharacterDetails";
+import { styled } from "styled-components";
+import Dark from "./Images/Dark.jpg";
+import Light from "./Images/Light.jpg";
+import { useThemeContext } from "./Context/Theme-context";
+import ImagePreloader from "./Components/ImagePreloader";
+
+const Main = styled.div`
+  margin: 0;
+  height: 100%;
+  width: 100%;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
 
 function App() {
+  const { themeIsLight } = useThemeContext();
+
+  useEffect(() => {
+    document.body.style.background = themeIsLight
+      ? `url(${Light}) `
+      : `url(${Dark})`;
+    document.body.style.color = themeIsLight ? "black" : "white";
```

Comment:
Maybe easier to just define 2 css classes that you add to document body based on light or dark modes.

Reply:
I made the change, yes, it is easier and the code seems more readable